seed/models/derived_columns.py

```diff
@@ -150,7 +152,7 @@ def evaluate(self, parameters=None):
             parameters = {}
 
         self._transformer.set_params(parameters)
-        return self._parser.parse(self._expression)
+        return self._parser.parse(self._expression)  # type: ignore[return-value]
```

Comment:
Great example of types not working well sometimes. mypy is unable to infer that the Tree has a decorator which evaluates the tree (in this case into a float)